### DISCUSSION TITLE
Live DTMF

### DISCUSSION THREAD

**Discussion by gavinsmith8181** [rtckit/eqivo#20] [Ideas]
Hey! thought about a nice idea which allow to send DTMF and a when it detects one, it will be sent over with call id.

I found myself looking at some customers which Sends DTMF a few times even whenever they aren't on GetDigits action, and i would love to save that info as well.

**Comment by gavinsmith8181** [comment]
Just found out it's possible on conference.

**Comment by cdosoftei** [comment]
Indeed, the `digitsMatch` attribute does that (as discussed in [this thread](https://github.com/rtckit/eqivo/discussions/21#discussioncomment-7668444)).
